extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/FilterBuildItem.java

```diff
@@ -11,6 +11,11 @@
  */
 public final class FilterBuildItem extends MultiBuildItem {
 
+    //predefined system priorities
+    public static final int AUTHENTICATION = 200;
+    public static final int CORS = 150;
```

Comment:
CORS should take precedence over AUTHENTICATION and AUTHORIZATION since the credentials may be wrong